b2share/modules/records/triggers.py

```diff
@@ -94,6 +90,11 @@ def set_record_owner(record, **kwargs):
     record['owner'] = str(current_user.id)
 
 
+def add_oai_id(record):
```

Comment:
Better to wait for https://github.com/EUDAT-B2SHARE/b2share/pull/983 to be ready then, before merging this one.